data/example-rdf/turtle/biobank.ttl

```diff
@@ -1,26 +1,52 @@
 @prefix : <http://purl.org/ejp-rd/metadata-model/v1/example-rdf/> .
-@prefix dcat:  <http://www.w3.org/ns/dcat#> .
-@prefix dct:   <http://purl.org/dc/terms/> .
-@prefix ejp:   <http://purl.org/ejp-rd/vocabulary/> .
-@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
-@prefix sio:  <http://semanticscience.org/resource/> .
-@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix lang: <http://id.loc.gov/vocabulary/iso639-1/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
 @prefix ordo: <http://www.orpha.net/ORDO/> .
+@prefix datacite: <http://purl.org/spar/datacite/> .
 
-:biobank a ejp:Biobank ;
-  dct:publisher :organisation ;        
-  dct:title "Austrian Bone Marrow Donor Registry" ;
-  dct:description "Donor registry";
-  ejp:populationCoverage  :population_coverage ;
-  dcat:theme  ordo:Orphanet_171895 ;
-  foaf:page <http://www.stammzellspende.at> .
 
-:organisation a foaf:Organisation ;
-  dct:title "Fonds Österreichisches Stammzellregister" ;
-  dct:spatial  :location .
+:collection
+  a obo:OBIB_0000616, dcat:Resouce;
+  dct:identifier "collection-id";
+  dct:title "Muscle Tissue Collection";
+  dct:description "Example collection of samples";
+  dct:alternative "MTC";
```

Comment:
not in the current model

Reply:
I think, there is a typo here:
dcat:Resouce;

should be dcat:Resource;

Reply:
I changed it in the new commit